### PR TITLE
Feature/Login page

### LINKED ISSUE #21: [Frontend] User login page
An user needs to be able to login to the platform.
Log in

The authentication is handled by an Identity Server. The login page is displayed outside of the frontend application and will redirect back to it once the login is successful. We need to apply the styles from the mockup to this page: https://github.com/code4romania/stam-acasa/blob/develop/backend/src/StamAcasa.IdentityServer/Views/Account/Login.cshtml

### PR DESCRIPTION
**Pull request by Dima000**
### What does it fix?

Closes https://github.com/code4romania/stam-acasa/issues/21

- Customize the Login page so that it looks like the provided design
- Add Bulma CSS framework to the IdentityServer resources (Bulma is used on FE for styling)
- Add `.idea` files to gitignore

### How has it been tested?
- Tested only manually 

### To Do
-  Need to replace the 'Username' field with 'Email' field, I could not do that because I'm not familiar with the BE technology

### CODE REVIEW

**Comment by surdu** [comment]
We already developed a [Login component on the UI](https://github.com/code4romania/stam-acasa/tree/develop/frontend/src/components/Login). Would it be complicated to expose the login as an api endpoint and use it from the frontend ?

**Comment by Dima000** [comment]
@surdu 
@RaduCStefanescu Created the GitHub issue(https://github.com/code4romania/stam-acasa/issues/21), maybe he can answer

**Comment by surdu** [comment]
For the sake of speed, we could move forward like this and add an improvement task for the future to refactor the login as api + ui

PS: sorry, now I've read the bottom of the ticket 😅 . We'll move forward like this!

**Review by surdu — APPROVED** [review]

File: backend/src/StamAcasa.IdentityServer/Views/Shared/_Layout.cshtml [hunk 1]
```diff
@@ -12,10 +12,11 @@
     <meta charset="utf-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>IdentityServer4</title>
+    <title>Stăm acasă</title>
     <link rel="icon" type="image/x-icon" href="~/favicon.ico" />
```
Comment (surdu) [inline backend/src/StamAcasa.IdentityServer/Views/Shared/_Layout.cshtml]: Can you please make sure that favicons are also set as [in here](https://github.com/code4romania/stam-acasa/blob/develop/frontend/public/index.html#L5-L17). Please remove `%PUBLIC_URL%` part in your case.

**Review by surdu — CHANGES_REQUESTED** [review]

File: backend/src/StamAcasa.IdentityServer/Views/Shared/_Layout.cshtml (hunk 1 above)
Comment (Dima000) [reply backend/src/StamAcasa.IdentityServer/Views/Shared/_Layout.cshtml]: Updated!

**Review by surdu — APPROVED** [review]

**Pull request merged**